haystack/management/commands/build_solr_schema.py

```diff
@@ -3,15 +3,21 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 from django.core.exceptions import ImproperlyConfigured
-from django.core.management.base import BaseCommand
+from django.core.management.base import BaseCommand,CommandError
 from django.template import Context, loader
+from django.conf import settings
 
 from haystack import connections, connection_router, constants
 from haystack.backends.solr_backend import SolrSearchBackend
 
-
+import pysolr
+import os
+import traceback
```

Comment:
I use `isort` to sort imports and split them into multiple sections (stdlib, third-party, this project). See 9f8c5833875e27e6e34526e2a3417b970d76e77a